src/index.html

```diff
@@ -22,6 +22,373 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="index.html">
+        <img
+          class="logo"
+          src="/src/images/logo.png"
+          alt="logo"
+        />
```

Comment:
The logo image path should be relative to the location of the HTML file. Since the `src` attribute starts with `/`, it indicates an absolute path from the root, which might not work as expected if the site is hosted in a subdirectory. Use `./images/logo.png` instead.